ui/components/CommitsTable.tsx

```diff
@@ -84,20 +84,23 @@ function CommitsTable({
         fields={[
           {
             label: "SHA",
+            displayLabel: "SHA",
             value: (row: Commit) => (
               <Link newTab href={row.url}>
                 {row.hash}
               </Link>
             ),
           },
           {
-            label: "Date",
+            label: "date",
+            displayLabel: "Date",
             value: (row: Commit) => timestamp(row.date),
           },
-          { label: "Message", value: "message" },
-          { label: "Author", value: "author" },
+          { label: "message", displayLabel: "Message", value: "message" },
+          { label: "author", displayLabel: "Author", value: "author" },
         ]}
         rows={commits.commits}
+        disablePagination
```

Comment:
How about the reverse? That way, we can "opt in" in a few places instead of opting out everywhere.